pyproject.toml

```diff
@@ -12,7 +11,151 @@ requires = [
     "pybind11>=2.10.0",  # If this comes out of sync with the version installed by Conan please update the version in CMakeLists
     "grpcio-tools",
 ]
+build-backend = "setuptools.build_meta"
+
+[project]
```

Comment:
This section should perhaps be the first section in this file.

Reply:
https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html

According to this, [build-system] is defined before [project]